### PR TITLE
black nav bar on image preview

### LINKED ISSUE #6242: preview image: white navigation bar

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #6242 

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [x] Tests written, or not not needed

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky afaik the navigationbar color can be defined v21+ so why is this qadded to a v27 style?

**Comment by jancborchardt** [comment]
It's black, but also half transparent in the image viewer just like the header bar, right? From the code "translucent" it looks like that is the case, can you confirm?

File: src/main/res/values-v21/styles.xml [hunk 1]
```diff
@@ -75,4 +75,9 @@
         <item name="android:windowBackground">@drawable/launch_screen</item>
         <item name="android:textColorHint">@color/secondary_text_color</item>
     </style>
+
+    <style name="Theme.ownCloud.Overlay" parent="Theme.ownCloud.OverlayBase">
```
Comment (tobiasKaminsky) [inline src/main/res/values-v21/styles.xml:79]: So on <v21 what is then used? Parent of Overlay? (never did this before :see_no_evil: )

**Comment by tobiasKaminsky** [comment]
> It's black, but also half transparent in the image viewer just like the header bar, right? From the code "translucent" it looks like that is the case, can you confirm?

You can try it yourself by installing the QA build :-)

**Comment by AndyScherzinger** [comment]
> You can try it yourself by installing the QA build :-)

which then leads to the test result that the bar isn't translucent... but black :/

**Comment by tobiasKaminsky** [comment]
I changed black to transparent, but then it is white…


When toggling fullscreen it becomes transparent:
 

@AndyScherzinger if you have time to fiddle around with this, it would be awesome!

**Comment by AndyScherzinger** [comment]
> if you have time to fiddle around with this, it would be awesome!

Not sure if I find the time @tobiasKaminsky so I'd vote for merging as is and opening a follow-up issue since this (partial) fix is still an improvement 👍

**Pull request merged**